### PR TITLE
Wizard refactor

### PR DESCRIPTION
**Pull request by Majkluss**

### CODE REVIEW

File: prusa/link/web/wizard.py [hunk 1]
```diff
@@ -116,13 +115,17 @@ def process_network(config):
...
+    app.wizard.connect_port = 0
+
     for option in config.options(CONNECT):
-        if option == 'hostname':
-            app.wizard.connect_hostname = connect['hostname']
-        if option == 'tls':
-            app.wizard.connect_tls = connect.getboolean('tls')
-        if option == 'port':
-            app.wizard.connect_port = connect['port']
+        # if option == 'hostname':
```
Comment (ondratu) [inline prusa/link/web/wizard.py:123]: What if i have another connect settings in `prusa_printer_settings.ini` which i want to restore. That is ok, to use my settings.

File: prusa/link/templates/_wizard.html [hunk 2]
```diff
@@ -4,25 +4,21 @@
     <div class="container">
 
 <div class="progress">
-    {%- if active == 'welcome' or active == 'auth' or active == 'printer' or active == 'connect' or active == 'finish'%}
-    <div class="progress-bar bg-warning {{ 'progress-bar-striped progress-bar-animated' if active == 'welcome' else '' }}" role="progressbar" style="width: 20%" aria-valuenow="20" aria-valuemin="0" aria-valuemax="100">
+    {%- if active == 'welcome' or active=='restore' or active == 'credentials' or active == 'printer' or active == 'finish'%}
```
[the remark this answers is not in the document]
Comment (Majkluss) [reply prusa/link/templates/_wizard.html:7]: Good point, I'll change it

File: prusa/link/templates/wizard_printer.html [hunk 3]
```diff
@@ -16,7 +16,7 @@ <h1>Setup Printer</h1>
             <div class="form-group row">
               <label for="name" class="col-sm-2 col-form-label">Name:</label>
               <div class="col-sm-10">
-                <input type="text" class="form-control {{ 'is-invalid' if errors.get('name') else '' }}" id="name" name="name" value="{{ wizard.printer_name or '' }}" required>
+                <input type="text" class="form-control {{ 'is-invalid' if errors.get('name') else '' }}" id="name" name="name" value="{{ wizard.printer_name or '' }}" placeholder="Original Prusa i3 MK3S+">
```
[the remark this answers is not in the document]
Comment (Majkluss) [reply prusa/link/templates/wizard_printer.html:19]: Oh I see, my original thought was to use only one placeholder, but I'll change it to dynamic type detecion

File: prusa/link/web/wizard.py (hunk 1 above)
Comment (Majkluss) [reply prusa/link/web/wizard.py:123]: Mistake, I'll fix it

File: prusa/link/templates/wizard.html [hunk 4]
```diff
@@ -4,75 +4,91 @@
 {% include "_header.html" %}
 {% set active="welcome" %}
 
-<div class="container">
+<div class="container" xmlns="http://www.w3.org/1999/html">
```
[the remark this answers is not in the document]
Comment (Majkluss) [reply prusa/link/templates/wizard.html:7]: Mistake, I'll fix it

**Review by ondratu — APPROVED** [review]

**Pull request merged**